apps/studio/Dockerfile

```diff
@@ -41,6 +41,9 @@ COPY --from=turbo /app/out/pnpm-lock.yaml ./
 # No need to clean cache because production uses standalone build
 RUN pnpm install --frozen-lockfile
 
+ARG BUILD_VERSION
+ENV STUDIO_VERSION="$BUILD_VERSION"
+
```

Comment:
You want to move this to line 29 so it's available on both dev and production images.

Reply:
pushed an update :)